challenge.js

```diff
@@ -26,7 +27,9 @@
  */
 
 // ANSWER
+They are not siblings because they do not share the same parent so I suppose they are not related.
```

Comment:
E is a grandchild of A. Drawing a family tree would definitely help make this apparent.